benchmarkplotter.d

```diff
@@ -0,0 +1,120 @@
+import std.stdio;
```

Comment:
needs shebang (for lazy people with rdmd)
```
#!/usr/bin/env rdmd
```

and maybe a short comment header explaining the program and author ;-)